proofs/fstar/extraction-edited/Libcrux.Kem.Kyber.Hash_functions.fst

```diff
@@ -18,12 +18,13 @@ let v_PRF (v_LEN: usize) (input: t_Slice u8) =
   admit(); // We assume that sha3_512 correctly implements H
   res
 
-let v_XOFx4 (v_K: usize) (input: t_Array (t_Array u8 (sz 34)) v_K) =
+let v_XOFx4 v_K (input: t_Array (t_Array u8 (sz 34)) v_K) =
+  assert (v v_K >= 2);
   let out:t_Array (t_Array u8 (sz 840)) v_K =
     Rust_primitives.Hax.repeat (Rust_primitives.Hax.repeat 0uy (sz 840) <: t_Array u8 (sz 840)) v_K
   in
   let out:t_Array (t_Array u8 (sz 840)) v_K =
-    if ~.(Libcrux_platform.simd256_support <: bool) || ~.false
+    if ~.(Libcrux_platform.simd256_support () <: bool) || ~.true
```

Comment:
This I think is now `~.(Libcrux_platform.simd256_support () <: bool)` without the `~.true` in `extraction`